src/fragments/Markdown.react.js

```diff
@@ -85,6 +88,10 @@ export default class DashMarkdown extends Component {
         const displayText =
             dedent && textProp ? this.dedent(textProp) : textProp;
 
+        const componentTransforms = {
+            DccLink: props => <DccLink {...props} />,
+        };
```

Comment:
Here we define the list of components we want to support are their name in markdown.